core/util/model/Logger.py

```diff
@@ -25,8 +25,8 @@ def logFatal(self, msg: str):
 		self.doLog(function='fatal', msg=msg)
 
 
-	def logWarning(self, msg: str):
-		self.doLog(function='warning', msg=msg, printStack=False)
+	def logWarning(self, msg: str, printStack: bool = False):
+		self.doLog(function='warning', msg=msg, printStack=printStack)
```

Comment:
Was that an old file? Because I added this yesterday?

Reply:
hm should have been the latest version from the alpha3 branch

Reply:
hm no it is on alpha3 ... No clue what I did then ^^

Reply:
Yeah, that's why I was asking, wondering if you are working on latest files

Reply:
well apparently I did forget to pull before creating the branch ;)

Reply:
I guess when I squash it in the end the changes from Logger will just disappear because I just changed it the way you did aswell